### PR TITLE
Add platelet.app Latest

### PR DESCRIPTION
**Pull request by amorist**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/platelet.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'platelet' do
+  version '0.0.13'
+  sha256 'aba7f4239ba3f34cf7bf13926c85c7b41afb192dc6647594cdf55e1fd989e7f6'
+
+  url "https://github.com/amorist/platelet/releases/download/v0.0.13/platelet-0.0.13.dmg"
+  name 'platelet'
```
Comment (vitorgalvao) [inline Casks/platelet.rb]:
```suggestion
  name 'Platelet'
  name 'けっしょうばん'
```

File: Casks/platelet.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask 'platelet' do
+  version '0.0.13'
+  sha256 'aba7f4239ba3f34cf7bf13926c85c7b41afb192dc6647594cdf55e1fd989e7f6'
+
+  url "https://github.com/amorist/platelet/releases/download/v0.0.13/platelet-0.0.13.dmg"
```
Comment (vitorgalvao) [inline Casks/platelet.rb]:
```suggestion
  url "https://github.com/amorist/platelet/releases/download/v#{version}/platelet-#{version}.dmg"
```

File: Casks/platelet.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  name 'platelet'
+  appcast 'https://github.com/amorist/platelet/releases.atom'
```
Comment (vitorgalvao) [inline Casks/platelet.rb]: This is fixed by the `brew cask style --fix`. Please make sure to follow the commands *before* checking the boxes.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by amorist** [comment]
Thanks.

**Pull request merged**